runtime/vm/CMakeLists.txt

```diff
@@ -21,20 +21,24 @@
 ################################################################################
 
 omr_add_tracegen(j9vm.tdf)
-add_library(j9vm SHARED
+
+if(OMR_OS_AIX AND (OMR_TOOLCONFIG STREQUAL "xlc"))
+	omr_remove_flags(CMAKE_CXX_FLAGS "-O3")
+	omr_remove_flags(CMAKE_C_FLAGS "-O3")
```

Comment:
In the UMA change, this only applies to the CXX_FLAGS.

Also, can this be merged into the other `if(OMR_OS_AIX AND (OMR_TOOLCONFIG STREQUAL "xlc"))` block?

Reply:
The problem occurs when re-applying the flag since the method we use doesn't discriminate c vs c++ sources, thus its easier to remove the flags everywhere, and apply them everywhere (other than the interpreter). Although, thinking it through, really the worst case scenario is that we add `-O3` twice on the command line, which shouldn't be an issue